.github/workflows/ci.yml

```
@@ -209,6 +207,39 @@ jobs:
           rm -rf ~/.m2/repository/org/apache/ozone/hdds*
           rm -rf ~/.m2/repository/org/apache/ozone/ozone*
         if: always()
+  basic-test:
```

```suggestion
  unit:
```

We also need to change `coverage` to depend on `unit` instead of `basic` (which no longer generates test coverage data).

---

> We also need to change `coverage` to depend on `unit` instead of `basic` (which no longer generates test coverage data).

Ok. done.